src/screens/ChannelListScreen/elements/Search.js

```diff
@@ -61,7 +62,7 @@ const styles = StyleSheet.create({
   wrapperSearch: {
     flex: 1,
     flexDirection: 'row',
-    backgroundColor: COLORS.lightgrey,
+    backgroundColor: '#F5F5F5',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The background color has been changed to a hardcoded value. It's better to use color constants for consistency and ease of updates.
```diff
-    backgroundColor: COLORS.lightgrey,
+    backgroundColor: '#F5F5F5',
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->